portality/bll/services/article.py

```diff
@@ -175,12 +193,18 @@ def create_article(self, article, account, duplicate_check=True, merge_duplicate
 
         # quickly validate that the article is acceptable - it must have a DOI and/or a fulltext
         # this raises an exception if the article is not acceptable, containing all the relevant validation details
-        self.is_acceptable(article)
+
+        try:
+            self.is_acceptable(article)
+        except Exception as e:
+            raise e
```

Comment:
isn't this equivalent to what it replaced? (apart from it's clearer that an exception is expected I suppose)

Reply:
Yes, it is. But I added that for clarity exactly. The `create_article` method is the "main" one and while looking at that we (or at least I) do not necessary check every single helper method. So I guess it is good no know from its level that it may raise an error.